pkg/apis/storage/validation/validation.go

```diff
@@ -237,6 +239,13 @@ func validateVolumeError(e *storage.VolumeError, fldPath *field.Path) field.Erro
 	if len(e.Message) > maxVolumeErrorMessageSize {
 		allErrs = append(allErrs, field.TooLong(fldPath.Child("message"), "" /*unused*/, maxAttachedVolumeMetadataSize))
 	}
+
+	if e.ErrorCode != nil && utilfeature.DefaultFeatureGate.Enabled(features.MutableCSINodeAllocatableCount) {
```

Comment:
strategy.go should drop the field, so we don't need the feature gate check.

Reply:
Makes sense, dropped.